util/election/mock/election.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package mock contains a mock MasterElection implementation.
```

Comment:
Also considered generated mocks (`//go:generate mockgen ...`), but it seemed that manually typed one better served my purposes, and also did not require `EXPECT` each and every call, but rather focus on the output of election algorithm.

Reply:
nit: I think this is more of a stub than a mock, maybe?

Reply:
Done.